pkgs/development/libraries/science/math/petsc/default.nix

```diff
@@ -68,6 +69,10 @@ stdenv.mkDerivation rec {
       ''}
       "--with-blas=1"
       "--with-lapack=1"
+      ${lib.optionalString withHypre ''
```

Comment:
Can you convert this to use configureFlags nix attribute instead of the bash variable?